RxJavaTemplate/app/src/androidTest/java/co/nimblehq/rxjava/ui/screens/home/HomeFragmentTest.kt

```diff
@@ -0,0 +1,50 @@
+package co.nimblehq.rxjava.ui.screens.home
+
+import androidx.test.filters.LargeTest
+import co.nimblehq.rxjava.domain.repository.ApiRepository
+import co.nimblehq.rxjava.ui.common.launchFragment
+import co.nimblehq.rxjava.ui.common.waitForApiThen
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import javax.inject.Inject
+
+@LargeTest
+@HiltAndroidTest
+class HomeFragmentTest {
+
+    @get:Rule
+    var hiltRule = HiltAndroidRule(this)
+
+    @Inject
+    lateinit var testApiRepository: ApiRepository
+
+    @Before
+    fun setup() {
+        hiltRule.inject()
+        launchFragment()
+    }
+
+    @Test
+    fun showUiCorrectly() {
```

Comment:
need to define a context. How about? 🙏 
```suggestion
    fun initData_showUiCorrectly() {
```